### ISSUE TITLE
Accordion heading text is blue on iOS 15

### ISSUE THREAD

**Issue by sam-wmca** [wmcadigital/wmn-design-system#892]
**Describe the bug**

**To Reproduce**
1. Go to [Find a Travel Centre](https://www.tfwm.org.uk/get-help/find-a-travel-centre/) on an iOS device
2. Scroll down to accordions

**Expected behaviour**
Accordion heading should be text colour.

**Screenshots**
![IMG_0121](https://user-images.githubusercontent.com/66072673/139104631-39b33810-b17b-4f75-b805-48bc4d3fc4d0.PNG)

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone 12, iPad]
 - OS: [e.g. iOS 15.1]
 - Browser: Safari
 - Version: 15.1

**Additional context**
This doesn't happen on desktop browsers, only iOS/iPad OS.

**Comment by houbly** [comment]
@sam-wmca is this still an issue? The accordions have been removed so if this is showing anywhere else let me know.

**Comment by sam-wmca** [comment]
@houbly this seems to have fixed itself, will close the ticket 🙂
